### DISCUSSION TITLE
When Tooltip is used in H5, the floating frame will disappear when clicked continuously.

### DISCUSSION THREAD

**Discussion by daidai-perfect** [chakra-ui/chakra-ui#8877] [General]
### Description

When Tooltip is used in H5, the floating frame will disappear when clicked continuously.

### Link to Reproduction

https://v2.chakra-ui.com/docs/components/tooltip  button

### Steps to reproduce

1. Go to 'https://v2.chakra-ui.com/docs/components/tooltip  H5'
2. Click on 'Button btn'
3. See error The ’Search places‘ floating box disappeared when clicked for the second time.

### Chakra UI Version

2.8.2

### Browser

谷歌 124.0.6367.202

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by daidai-perfect** [comment]
Continuous clicking on the web page will cause the same problem

**Comment by segunadebayo** [comment]
v2 is in maintenance, so we are not changing this. Stay on 2.8.x, or migrate to v3 if you can.
